test/integration/test_journal_entries.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import pytest
+import json
+import os
+
+logger = logging.getLogger(__name__)
+
+def test_get(nc):
+    data = next(nc.journal_entries.get_all_generator())
+    logger.debug('data = %s', data)
+    assert data, 'get all generator didnt work'
```

Comment:
can you assert more things about the result to ensure that it works as expected?